Dockerfile

```diff
@@ -173,16 +173,21 @@ RUN  --mount=target=/root/.cache,type=cache \
 
 # Based on https://github.com/containers/fuse-overlayfs/blob/v0.7.6/Dockerfile.static.ubuntu .
 # We can't use Alpine here because Alpine does not provide an apk package for libfuse3.a .
-FROM debian:10 AS fuse-overlayfs
+FROM --platform=$BUILDPLATFORM debian:10 AS fuse-overlayfs
 RUN apt-get update && \
   apt-get install --no-install-recommends -y \
-  git ca-certificates libc6-dev gcc make automake autoconf pkgconf libfuse3-dev file
+  git ca-certificates libc6-dev gcc make automake autoconf pkgconf libfuse3-dev file curl
+ARG TARGETPLATFORM
+# https://github.com/tonistiigi/binfmt/pull/1
```

Comment:
I have merged this. Although I might force-push the repo so you can keep it linked to your fork for now. Overall should move this to `tonistiigi/xx`